charts/csm-application-mobility/Chart.yaml

```diff
@@ -20,6 +20,6 @@ dependencies:
   repository: https://charts.jetstack.io
   condition: cert-manager.enabled
 - name: velero
-  version: 2.29.8
+  version: 4.1.4
```

Comment:
With these changes to the helm chart, we'll also have to bump up the version of the helm chart (lines 9 and 15 above). Similar to other comments below, this should likely be 1.0.0